src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
```

Comment:
[CODE STYLE] - Consider using a more descriptive variable name than `styles` for the result of the transformation. Since `styles` represents an object with CSS properties and their values, a name like `styleObject` or `cssProperties` would enhance readability.

Reply:
Good point 👍 
Change it